### ISSUE TITLE
Use on Gratisdata, a non-WMF wiki

### ISSUE THREAD

**Issue by ugochimobi** [Dhairya3124/Wikidata-Complete-Gadget#35]
Hello, @Dhairya3124 I hope you're doing great, I just want to let you guys know that, this gadget is a very wonderful tool, at least I've used it several times on Wikidata, and it's amazing using it.
Thanks to all the dev team members.

Can this gadget be used on a non-WMF wiki? it's [Gratisdata ](https://gratisdata.miraheze.org/wiki/)and it's also a wiki like Wikidata, which uses the same software (MediaWiki, Wikibase) as Wikidata.

I am very much interested in hearing from you, thank you for your anticipated cooperation in this matter.

**Comment by ugochimobi** [comment]
@anbo-de I appreciate your reply, thank you! \o/

I would be very happy and willing to give intermediate feedback to the upcoming implementation.

Nevertheless, I have actually installed the code on Gratisdata as a gadget, you can find the [Gadget source here](https://gratisdata.miraheze.org/w/index.php?title=MediaWiki:Gadget-GratisdataComplete.js), although you have to create an account so that you can enable the gadget on your user preference. But it currently doesn't work, I have no idea why though, because, I have done the necessary modifications except this `qanswer-svc3.univ-st-etiene.fr` URL.
You can view the [GitHub mirror here
](https://github.com/ugochimobi/Wikibase-Complete-Gadget/blob/main/WikidataComplete-Gadget.js)

Is there another API that can be used to return approvable facts? or Can we not use this current one for Gratisdata?

I deeply appreciate your time for replying and I'd love to hear from you soon.

**Comment by D063520** [comment]
Does the entity have a related text? Where would you like to extract the knwoledge from. Also a pointer for your wikibase:

https://github.com/the-qa-company/WikibaseSync

**Comment by ugochimobi** [comment]
@D063520 Yes, the entities' related text should probably come from English Wikipedia, and also Gratispaideia (a sister to Gratisdata) and any other open-source that can possibly be used.

**Comment by ugochimobi** [comment]
Also, https://github.com/the-qa-company/WikibaseSync can this be used to import entities from Wikidata?

**Comment by Dhairya3124** [comment]
> Also, thank you for sharing this; https://github.com/the-qa-company/WikibaseSync can this be used to import entities from Wikidata?

Yes

**Comment by D063520** [comment]
Yes you can use it also when you already have some entities ....

**Comment by ugochimobi** [comment]
> @D063520 Yes, the entities' related text should probably come from English Wikipedia, and also Gratispaideia (a sister to Gratisdata) and any other open-source that can possibly be used.

@D063520 many thanks :) 
I gladly await further response on this from maybe @anbo-de because I saw that he mentioned his comment in the Proposal issue on Wikimedia phabricator.

**Comment by anbo-de** [comment]
@ugochimobi We discussed the issue again in the team. As is we have limited time in the current project, we decided to postpone this extension of the current functionality and focus on the tasks we promised in the Google Summer of Code program. However, we will later re-evaluate. 

However, if you want to extend the functionality by yourself, you are free to integrate it into the source code of this project.

**Comment by ugochimobi** [comment]
> @ugochimobi We discussed the issue again in the team. As is we have limited time in the current project, we decided to postpone this extension of the current functionality and focus on the tasks we promised in the Google Summer of Code program. However, we will later re-evaluate.
> 
> However, if you want to extend the functionality by yourself, you are free to integrate it into the source code of this project.

Hi thanks, is it possible to integrate it (Gratisdata so to say) into the source code here?

**Comment by ugochimobi** [comment]
> @ugochimobi We discussed the issue again in the team. As is we have limited time in the current project, we decided to postpone this extension of the current functionality and focus on the tasks we promised in the Google Summer of Code program. However, we will later re-evaluate.
> 
> However, if you want to extend the functionality by yourself, you are free to integrate it into the source code of this project.

Hello @anbo-de, I hope you have been doing great since our last conversation:)
I want to ask if this can be reevaluated? Or reconsidered?

**Comment by ugochimobi** [comment]
Hello @anbo-de @D063520 
I hope you're doing great.
I want to further this conversation. Is it now possible to use this gadget on an external wikidata-like site like gratisdata.miraheze.org?

**Comment by D063520** [comment]
Hi @ugochimobi: I think this will not be possible in the near future! We are busy on other pojects unfortunately! Salut Dennis
